### PR TITLE
EZP-31355: When browsing in UDW children view is blocked by content meta preview

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-31355

### CODE REVIEW

File: src/modules/universal-discovery/components/finder/finder.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 import { LoadedLocationsMapContext } from '../../universal.discovery.module';
 
 const Finder = () => {
     const [loadedLocationsMap, dispatchLoadedLocationsAction] = useContext(LoadedLocationsMapContext);
+    const finderRef = useRef(null);
+
+    useEffect(() => {
+        const container = finderRef.current;
+
+        if (!container) {
```
Comment (GrabowskiM) [inline src/modules/universal-discovery/components/finder/finder.js]: Is this if needed? useEffect is called each time after page is rendered, so this `<div>` will be always created.

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by OstafinL — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA Approved on eZ Platform OSS v3.0 master & eZ Platform EE v3.0 master with diff. Travis failure unrelated.

**Comment by lserwatka** [comment]
@dew326 could you rebase?

**Comment by dew326** [comment]
@lserwatka done

**Pull request merged**